src/utils/date.ts

```diff
@@ -0,0 +1,16 @@
+import { DateTime } from "luxon";
+
+export interface DateAsStrings {
```

Comment:
Rather than define a new structure here, could you maybe (re-) use the `InputDate` definition in data.types.model.ts?

Reply:
yes, didn't spot that, looks the same so could re-use it